shared/src/test/java/org/odk/collect/shared/QuickCheck.kt

```diff
@@ -0,0 +1,16 @@
+package org.odk.collect.shared
+
+/**
+ * Simple helper to allow writing neater [QuickCheck](https://en.wikipedia.org/wiki/QuickCheck)
+ * style tests.
+ */
+fun <Input, Output> ((Input) -> Output).quickCheck(
```

Comment:
I did have a quick look at using an existing framework (https://github.com/pholser/junit-quickcheck for example), but it felt like we didn't really need anything so heavy yet. Maybe if we end up writing more of these kinds of tests down the line!